src/dashboard/Dashboard.tsx

```diff
@@ -67,7 +67,11 @@ const Dashboard = (): ReactElement => {
   ];
 
   const disconnect = (): void => {
-    history.push(Path.CONNECT_TO_REMOTE);
+    const nextPath =
+      (window as any).electron.platform() === "win32"
```

Comment:
Minor nit that can be done in future PRs - if we need to use this logic again let's extract it to `isWindows()` utils function. It's currently being used in 2 places.